### DISCUSSION TITLE
Error al crear archivo vue

### DISCUSSION THREAD

**Discussion by JoseAnayaR** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#509] [Show and tell]
![image](https://user-images.githubusercontent.com/79932287/159855870-54ac1fc0-6f3e-449d-8dd0-6a4477adced5.png)

Hola, desde que creo el nuevo vue como lo hace el MC me da este error, no he movido nada de nada desde que lo crea me lanza ese error :(

**Comment by Burstniart** [comment]
Es por un elemento que se crea en HelloWorld.vue, nada mas ve al archivo y modifica el html dentro de la etiqueta template, todo tu html debe estar dentro de las etiquetas template por componente para que no te arroje ese error

**Reply by JoseAnayaR** [reply]
Pero según yo si esta todo bien, como te digo desde el momento de crearlo me aparece ese error mira aquí te dejo el archivo con el html y todo esta normal a mi parecer 
![image](https://user-images.githubusercontent.com/79932287/160182466-87ce7bdc-bbd9-4914-80e2-9287836e7413.png)
